sma-em/sensors.py

```diff
@@ -205,7 +206,8 @@ def startup():
                 "u1:min",
             ],
             THRESHOLD: 80,
-            DEBUG: 0,
+            RECONNECT_INTERVAL: 0,
```

Comment:
Not important in production, but used this in local testing

```suggestion
            RECONNECT_INTERVAL: 86400,
```